rel-eng/tito.props

```diff
@@ -133,6 +133,7 @@ whitelist = foreman
   rubygem-gridster-rails
   rubygem-hammer_cli
   rubygem-hammer_cli_foreman
+  rubygem-foreman_maintain
```

Comment:
Ahem, I thought I saw it in `foreman-nightly-nonscl-rhel7` last time. But I added it this time. Please verify :)